index.bs

```diff
@@ -456,6 +456,10 @@ Insert the following steps in the <a spec="fetch">HTTP-network fetch</a> algorit
     run [=process a fetch storage access for bounce tracking mitigations=]
     given <var ignore>request</var>.
 
+Note: This spec is written for the case where third-party cookies are blocked. Any cookies written by embedded third-party sites are partitioned.
```

Comment:
These two sentences look like they contradict each other. I might be missing the context, but how can a cookie be both blocked and partitioned?